application/analyst/services/voice_drift_service.py

```
@@ -107,8 +107,8 @@ def score_chapter(
             adjective_density=metrics["adjective_density"],
             avg_sentence_length=metrics["avg_sentence_length"],
             sentence_count=metrics["sentence_count"],
-            # 存 None 而非 0.0，避免漂移检测误判
-            similarity_score=similarity,
+            # 无指纹基准时存 0.0（NOT NULL 约束），业务层通过 fingerprint 是否存在来区分
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)